src/utils/webrtc/webrtc.js

```diff
@@ -951,6 +951,33 @@ export default function initWebRTC(signaling, _callParticipantCollection, _local
 		}
 	})
 
+	webrtc.on('sendToAll', function(messageType, payload) {
+		const sessionIdsForParticipantsWithPeers = webrtc.webrtc.peers.map(peer => {
+			return peer.id
+		})
+
+		// "webrtc.sendToAll" only sends the signaling message to participants
+		// for which there is a Peer object. Therefore the message needs to be
+		// explicitly sent here too to participants without audio and video.
+		for (const sessionId in usersInCallMapping) {
```

Comment:
does this work with sip callees?

Reply:
I do not know for sure. As far as I can tell `usersInCallMapping` should contain also the sessions of SIP participants, because there seem to be [a `CallParticipantModel` object for each SIP participant](https://github.com/nextcloud/spreed/blob/aac3602ac34f146cd52230d2eee281ba7e4d4d50/src/utils/webrtc/webrtc.js#L448), and [the `CallParticipantModels` are indirectly created](https://github.com/nextcloud/spreed/blob/aac3602ac34f146cd52230d2eee281ba7e4d4d50/src/utils/webrtc/webrtc.js#L248) from [the sessions in `usersInCallMapping`](https://github.com/nextcloud/spreed/blob/aac3602ac34f146cd52230d2eee281ba7e4d4d50/src/utils/webrtc/webrtc.js#L438). If there is a Peer object for those participants this change would not affect them, and if there is not they should start now to receive the signaling messages (as I assume that the `inCall` flag is set for them). But if you could test it that would be great :-)